test/test.ts

```diff
@@ -0,0 +1,49 @@
+/// <reference path="../d/mocha.d.ts" />
+/// <reference path="../d/chai.d.ts" />
+/// <reference path="../d/jsdom.d.ts" />
+/// <reference path="../d/d3.d.ts" />
```

Comment:
Thanks.

I would like to see usage of typing in this repository. Unfortunately we are using slightly customized version d3 definitions since we ran into issue with version from definitely typed. Anyway I know this and also our private repository with charting library (also TS) can be improved especially when it comes to usage of declarations. We already have card in backlog  so hopefully it won't take so long to solve it. Hope it makes sense.